Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -199,13 +214,17 @@ struct atom_wrapper {
              "      Defaults to 0.\n")
         .def("GetNumImplicitHs", &Atom::getNumImplicitHs, python::args("self"),
              "Returns the total number of implicit Hs on the atom.\n")
-
-        .def("GetExplicitValence", &Atom::getExplicitValence,
-             python::args("self"),
-             "Returns the explicit valence of the atom.\n")
-        .def("GetImplicitValence", &Atom::getImplicitValence,
-             python::args("self"),
-             "Returns the number of implicit Hs on the atom.\n")
+        .def(
+            "GetExplicitValence", &getExplicitValenceHelper,
+            python::args("self"),
+            "DEPRECATED, please use GetValence(True) instead.\nReturns the explicit valence of the atom.\n")
+        .def(
+            "GetImplicitValence", &getImplicitValenceHelper,
+            python::args("self"),
+            "DEPRECATED, please use GetValence(False) instead.\nReturns the number of implicit Hs on the atom.\n")
+        .def("GetValence", &Atom::getValence,
+             (python::args("self"), python::args("getExplicit")),
```

Comment:
I can't figure out how to make a keyword-only argument in boost::python. Can you?

Reply:
> I can't figure out how to make a keyword-only argument in boost::python. Can you?

I've never looked, but I'm not a fan of those anyway